packages/localizations/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
We shouldn't forget to mark this package as side-effect free, see
https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

(Reading the whole tree-shaking section is worth it)